backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +688,66 @@ def test_cancel_workshop_attendance(self):
         self.assertIsNone(self.card.workshop_attendance)
         self.assertEqual(WorkshopAttendance.objects.count(), 0)
         self.assertEqual(self.card.status, AgileCard.READY)
+
+
+class ReviewerIdsSinceLatestReviewRequest(TestCase):
+
+    def setUp(self):
+        self.card = factories.AgileCardFactory(
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.project = self.card.recruit_project
+        self.user = self.card.assignees.first()
+        self.assertIsNotNone(self.card.assignees)
+        self.assertIsNotNone(self.card.recruit_project)
+        self.assertEqual(self.card.status, AgileCard.IN_PROGRESS)
+        self.assertEqual(self.project.content_item, self.card.content_item)
+
+    def test_finding_latest_reviewer_ids(self):
```

Comment:
add one more test here to show that the right reviews count. 

Eg: make 2 projects such that:

```
len(project1.get_users_that_reviewed_since_last_review_request()) > 0
len(project2.get_users_that_reviewed_since_last_review_request()) == 0
```